manifests/qpid.pp

```diff
@@ -10,31 +10,50 @@
   String $interface = 'lo',
   Integer[0, 5000] $wcache_page_size = 4,
 ) {
-  include certs
-  include certs::qpid
   include katello::params
 
   $agent_event_queue_name = $katello::params::agent_event_queue_name
```

Comment:
NIT: I think this should be moved under the if block (the one further down at line 50) where it is used

Reply:
This is used inside the `qpid_acls.acl` template.